docs/components/concepts/processes.md

```diff
@@ -2,6 +2,7 @@
 id: processes
 title: "Processes"
 description: "Processes are flowchart-like blueprints that define the orchestration of tasks."
+displayed_sidebar: Components
```

Comment:
I'm not exactly sure _why_ these need to be added.

The effect of not adding it is that _no_ sidebar shows up for this doc. 

[From the docs](https://docusaurus.io/docs/sidebar/multiple-sidebars#sidebar-association), it sounds like this property is needed to narrow down which "sidebar" to use when there's ambiguity. I initially thought "oh that's because I have two sidebars files with all the same sidebars in them", but even then, specifying "Components" doesn't tell it _which_ Components do use, since it's defined in both sidebars files. 

In the end I just said 🤷 and slapped these in, figuring it wouldn't be a terrible amount of work to add these to every doc -- potentially automatable, even.